src/vaadin-grid-tree-toggle.html

```diff
@@ -194,7 +194,7 @@
       /** @private */
       _levelChanged(level) {
         const value = Number(level).toString();
-        this.style['---level'] = value;
+        this.updateStyles({'---level': value});
```

Comment:
Can this change be tested?

Reply:
I added a test for this change.

Reply:
nit: it merely tests that the width gets synchronously changed. The width changing asynchronously isn't a bug per se. The bug, in this case, is that `recalculateColumnWidths()` doesn't wait until a tree-toggle's `level` change gets the styles properly applied but executes too soon.

Ideally, the test should verify that the bug caused by the width changing asynchronously, got fixed by the change = a test that adds a tree-toggle to a cell, applies a level-value for it, and invokes `recalculateColumnWidths()`.

Not a blocker for this PR though.